tests/test_infer.py

```diff
@@ -157,6 +157,183 @@ def test_hypotest_return_expected_set(tmpdir, hypotest_args, test_stat):
     assert check_uniform_type(result[3])
 
 
+@pytest.mark.parametrize(
```

Comment:
a lot of tests here are asserting a lot of extra behavior that's not needed as it's covered by other tests. Just try to keep it to only one or two assertions per test if possible. The test should only be looking at the calculators, so it shouldn't care about the other parts.

Reply:
Done. Thanks for reviewing! I took out the asserts about other return values, and collapsed the series of tests into one parameterized one. Let me know if its still asserting too much.